### ISSUE TITLE
Implement delayed transaction (unfreezing)

### ISSUE THREAD

**Issue by Geod24** [bosnet/sebak#309]
When #307 is done, we need a mean to delay a state change.
Basically, when a block is externalized, we should have a check for actions that have been delayed until this block. This will allow us to implement unfreezing efficiently.

**Comment by soonkuk** [comment]
I think both records of request for unfreezing and real unfreezing should be stored in blockchain.

**Comment by kyehyukahn** [comment]
Implicitly delayed tx may cause unexpected problems. So if we decide the delayed action, we need to add the fact about the delaying action on a blockchain.

**Comment by Geod24** [comment]
@soonkuk: Yes, we agree on this. The initial transaction is the request for unfreezing, then after the timeout the proposer generates the actual state change.

**Comment by soonkuk** [comment]
I imagined how about using delayed transaction for the coin issuance.
If delayed transaction make series action periodically, I think it works same as coin issuance.
